resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -165,6 +165,16 @@ private[spark] object Config extends Logging {
       .checkValue(_ <= 1048576, "Must have at most 1048576 bytes")
       .createWithDefault(1048576) // 1.0 MiB
 
+  val SPARK_CONF_DIR_CONFIG_MAP_NAME =
+    ConfigBuilder("spark.kubernetes.sparkConfDir.configMapName")
```

Comment:
I was confused initially because this configuration requires a user-provided config name.

Can we simply this new feature by simply controlled by a boolean configuration instead of a random string name?